manticore/native/memory.py

```diff
@@ -26,12 +26,13 @@
 logger = logging.getLogger(__name__)
 
 consts = config.get_group("native")
-consts.add(
-    "fast_crash",
-    default=False,
-    description="If True, throws a memory safety error if ANY concretization of a pointer is"
-    " out of bounds. Otherwise, forks into valid and invalid memory access states.",
-)
+if "fast_crash" not in consts:
+    consts.add(
+        "fast_crash",
+        default=False,
+        description="If True, throws a memory safety error if ANY concretization of a pointer is"
+        " out of bounds. Otherwise, forks into valid and invalid memory access states.",
+    )
```

Comment:
I think this might not be needed anymore after https://github.com/trailofbits/manticore/commit/dd57db819b68dcf6e1752ed8f61800165818e5d2

Reply:
Removed with tests passing!